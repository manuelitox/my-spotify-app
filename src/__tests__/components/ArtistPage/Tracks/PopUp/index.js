import React from 'react'
import { shallow } from 'enzyme'

import PopUpTopTracks from 'components/ArtistPage/Tracks/PopUp/'

describe('Component | PopUpTopTracks', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <PopUpTopTracks togglePopUp={ jest.fn() } />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})