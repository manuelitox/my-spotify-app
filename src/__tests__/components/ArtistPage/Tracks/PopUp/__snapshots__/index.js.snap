// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | PopUpTopTracks should render a component 1`] = `
<OnClickOutside(PopUpContainer)
  eventTypes={
    Array [
      "mousedown",
      "touchstart",
    ]
  }
  excludeScrollbar={false}
  isOpen={true}
  outsideClickIgnoreClass="ignore-react-onclickoutside"
  preventDefault={false}
  stopPropagation={false}
  togglePopUp={[MockFunction]}
>
  <Title
    color="#262D45"
    insidePopUp={true}
  >
    Top Tracks
  </Title>
  <styled.div>
    <Track
      duration={354320}
      isPopUpVersion={true}
      key="123asd"
      name="song name"
      previewUrl="path/to/preview/url"
      songIsAvailable={true}
    />
  </styled.div>
</OnClickOutside(PopUpContainer)>
`;
